pkg/controllers/node/termination/terminator/terminator.go

```diff
@@ -68,23 +79,27 @@ func (t *Terminator) Cordon(ctx context.Context, node *v1.Node) error {
 // https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
 func (t *Terminator) Drain(ctx context.Context, node *v1.Node) error {
 	// Get evictable pods
-	pods, err := t.getPods(ctx, node)
-	if err != nil {
-		return fmt.Errorf("listing pods for node, %w", err)
+	pods := &v1.PodList{}
```

Comment:
Listing pods doesn't work because it excludes DaemonSet pods? Should we create a function that includes the daemonset pods (or even an option) so that we don't include terminating pods in this calculation here?

Reply:
Listing pods does list DaemonSet pods.